Automattic/wp-calypso
# Typography: Switch font-weight: bold to 600

#### Changes proposed in this Pull Request

* Update instances of font-weight: bold to font-weight: 600 to align with typographic standards for sans-serif fonts across Calypso

**Visuals (not exhaustive)**

| Before | After |
| ------ | ------- |
| <img width="673" alt="Screen Shot 2020-05-18 at 12 45 57 PM" src="https://user-images.githubusercontent.com/2124984/82485992-399e1280-9aaa-11ea-83a1-2c2f812b8a29.png"> | <img width="685" alt="Screen Shot 2020-05-18 at 12 45 06 PM" src="https://user-images.githubusercontent.com/2124984/82486003-3dca3000-9aaa-11ea-87f7-eea028912781.png"> |
| <img width="380" alt="Screen Shot 2020-05-20 at 12 58 53 PM" src="https://user-images.githubusercontent.com/2124984/82486057-55091d80-9aaa-11ea-9ccc-5decec7f46d6.png"> | <img width="375" alt="Screen Shot 2020-05-20 at 12 58 42 PM" src="https://user-images.githubusercontent.com/2124984/82486070-5a666800-9aaa-11ea-8b5c-ce699f717ac4.png"> |
| <img width="380" alt="Screen Shot 2020-05-20 at 1 01 18 PM" src="https://user-images.githubusercontent.com/2124984/82486105-65b99380-9aaa-11ea-87cb-c55136518560.png"> | <img width="396" alt="Screen Shot 2020-05-20 at 1 01 06 PM" src="https://user-images.githubusercontent.com/2124984/82486115-694d1a80-9aaa-11ea-8f8d-9ee60dcbecc8.png"> |
| <img width="746" alt="Screen Shot 2020-05-20 at 2 13 06 PM" src="https://user-images.githubusercontent.com/2124984/82486155-766a0980-9aaa-11ea-9c9d-2afbfc8ffb0f.png"> | <img width="737" alt="Screen Shot 2020-05-20 at 2 12 45 PM" src="https://user-images.githubusercontent.com/2124984/82486165-79fd9080-9aaa-11ea-876a-03b399ee319f.png"> |
| <img width="739" alt="Screen Shot 2020-05-20 at 2 19 52 PM" src="https://user-images.githubusercontent.com/2124984/82486225-884bac80-9aaa-11ea-99f5-4aa6eafe9d22.png"> | <img width="738" alt="Screen Shot 2020-05-20 at 2 19 42 PM" src="https://user-images.githubusercontent.com/2124984/82486238-8c77ca00-9aaa-11ea-90bd-77789fbd54e3.png"> |

#### Testing instructions

* Switch to this PR
* Look for instances of bold text across WP.com; font weight should be `600` rather than `bold` or `700`. Make sure nothing breaks visually as a result of this change.
